schema/can.proto

```diff
@@ -58,12 +58,14 @@ message CanMsg {
     LIGHTS_FRONT = 5;
     LIGHTS_REAR = 6;
     MOTOR_CONTROLLER = 7;
-    DRIVER_CONTROLS = 8;
+    DRIVER_CONTROLS_CENTER_CONSOLE = 8;
     DRIVER_DISPLAY = 9;
```

Comment:
Technically the `DRIVER_DISPLAY` board is now the `CENTER_CONSOLE` board, so you _could_ just take over that Device ID..